ducc-sys/src/bindings.rs

```diff
@@ -1,13 +1,12 @@
 /* automatically generated by rust-bindgen */
 
-pub const DUK_VERSION: u32 = 20201;
-pub const DUK_GIT_COMMIT: &'static [u8; 41usize] = b"25420e773c5fbc50d5b46bf487fc45717e35b94f\0";
-pub const DUK_GIT_DESCRIBE: &'static [u8; 7usize] = b"v2.2.1\0";
-pub const DUK_GIT_BRANCH: &'static [u8; 17usize] = b"v2.2-maintenance\0";
-pub const DUK_USE_OS_STRING: &'static [u8; 6usize] = b"linux\0";
+pub const DUK_VERSION: u32 = 20500;
+pub const DUK_GIT_COMMIT: &'static [u8; 41usize] = b"6001888049cb42656f8649db020e804bcdeca6a7\0";
+pub const DUK_GIT_DESCRIBE: &'static [u8; 7usize] = b"v2.5.0\0";
+pub const DUK_GIT_BRANCH: &'static [u8; 7usize] = b"master\0";
+pub const DUK_USE_OS_STRING: &'static [u8; 4usize] = b"osx\0";
```

Comment:
It's funny that these (+ int MIN/MAX constants below) leak here, but I reckon it's not too harmful.